python/python_fastapi_server/routers/netcdf_to_covjson.py

```diff
@@ -264,19 +305,23 @@ def netcdf_to_covjson(netcdfdataset) -> Coverage:
         "TIME=2023-03-23T09:00:00Z"
     )
 
+    QUERYSTRING = "dataset=netcdf_5d&service=wcs&version=1.1.1&request=getcoverage&format=NetCDF4&crs=EPSG:4326&coverage=data&bbox=4.603,51.27,6.274,52.624&time=2017-01-01T00:00Z/2017-01-01T00:30Z&elevation=1000/9000"
+
     WCSGETCOVERAGEURL = SERVICE + QUERYSTRING
 
     # Get a NetCDF file as dataset
     response = requests.get(WCSGETCOVERAGEURL, timeout=60)
 
     print(response.status_code)
+    with open("/tmp/klad.nc", "wb") as f:
```

Comment:
Probably remove?

Reply:
Or make a bit nicer. I see it is meant for testing purposes.

Reply:
removed